### DISCUSSION TITLE
Suporte a Vida #2

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#307] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/tirinhas/suporte-a-vida-2/

**Comment by one-eyed-willy** [comment]
A geração Nutella não sabe o que é ter problema com download:



![null](<https://i.postimg.cc/m2C21j5D/Bq9fo35-CMAA0y-Eq.jpg>)

**Comment by brunoaugusto** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [sábado, 30 de abril de 2022](<https://www.vish.blog.br/imagens/tirinhas/suporte-a-vida-2/#comment_1114003243>):

> A geração Nutella não sabe o que é ter problema com download:[![null](<https://camo.githubusercontent.com/c91fbedfa18c557ef0e19bf3f085b3d833c9d856583a1f02dcd8b9e2468da773/68747470733a2f2f692e706f7374696d672e63632f6d324332316a35442f427139666f33352d434d414130792d45712e6a7067>)](<https://camo.githubusercontent.com/c91fbedfa18c557ef0e19bf3f085b3d833c9d856583a1f02dcd8b9e2468da773/68747470733a2f2f692e706f7374696d672e63632f6d324332316a35442f427139666f33352d434d414130792d45712e6a7067>)

![null](<https://i.postimg.cc/wB5ZJfK6/bane-internet-lenta.jpg>)
